deploy/playbooks/02_nginx.yml

```
@@ -4,22 +4,28 @@
   tasks:
     - name: Copy nginx configuration file
       ansible.builtin.template:
-        src: ../templates/nginx/nginx.conf.j2
+        src: nginx/nginx.conf.j2
         dest: ./nginx.conf
+        mode: "0644"
 
     - name: Create a server Makefile (for nginx) to manage on-server tasks
       ansible.builtin.template:
-        src: ../templates/nginx/Makefile.nginx.j2
+        src: nginx/Makefile.nginx.j2
         dest: ./Makefile
+        mode: "0644"
```

```suggestion
        mode: "0640"
```
No reason for other to read this.